python_src/src/lamp_py/ingestion/convert_gtfs_rt.py

```diff
@@ -330,12 +367,14 @@ def move_s3_files(self) -> None:
         move archive and error files to their respective s3 buckets.
         """
         if len(self.error_files) > 0:
+            self.moved_files += self.error_files
             self.error_files = move_s3_objects(
                 self.error_files,
                 os.path.join(os.environ["ERROR_BUCKET"], DEFAULT_S3_PREFIX),
             )
 
         if len(self.archive_files) > 0:
+            self.moved_files += self.archive_files
```

Comment:
add the error and archive files to the moved files. then this list is returned after conversion.